apps/api/src/request/repository.js

```diff
@@ -43,7 +43,7 @@ const getFilteredQuery = (client, filters, sort, user) => {
         .where(restFilters);
 
     if (delivery_postal_code) {
-        query.andWhere('delivery_postal_code', 'LIKE', `%${delivery_postal_code}`);
+        query.andWhere('delivery_postal_code', 'LIKE', `%${delivery_postal_code}%`);
```

Comment:
```suggestion
        query.andWhere('delivery_postal_code', 'LIKE', `%${deliveryPostalCode}`);
```

Reply:
and from line 33 to 36 of the same files, filter's name must be transform in camelCase too.

Reply:
I'm not agree with `%${deliveryPostalCode}` because with this, a search like '14' will not return all postalCode start with '14'.
I prefer `%${deliveryPostalCode}%` or, at least,  `${deliveryPostalCode}%`